Exception during training: Expecting value: line 5 column 20 (char 85)
Traceback (most recent call last):
  File "/opt/program/train", line 64, in train
    loaded_params = json.load(tc)
  File "/root/miniconda3/envs/daal4py/lib/python3.6/json/__init__.py", line 299, in load
    parse_constant=parse_constant, object_pairs_hook=object_pairs_hook, **kw)
  File "/root/miniconda3/envs/daal4py/lib/python3.6/json/__init__.py", line 354, in loads
    return _default_decoder.decode(s)
  File "/root/miniconda3/envs/daal4py/lib/python3.6/json/decoder.py", line 339, in decode
    obj, end = self.raw_decode(s, idx=_w(s, 0).end())
  File "/root/miniconda3/envs/daal4py/lib/python3.6/json/decoder.py", line 357, in raw_decode
    raise JSONDecodeError("Expecting value", s, err.value) from None
json.decoder.JSONDecodeError: Expecting value: line 5 column 20 (char 85)
